testing/load-aware/test.py

```diff
@@ -74,29 +74,29 @@ def _run_test(test_artifact_dir_p):
     """
 
     next_count = env.next_artifact_index()
-    with env.TempArtifactDir(env.ARTIFACT_DIR / f"{next_count:03d}__dummy_test"):
+    with env.TempArtifactDir(env.ARTIFACT_DIR / f"{next_count:03d}__load_aware_scale_test"):
         test_artifact_dir_p[0] = env.ARTIFACT_DIR
 
         with open(env.ARTIFACT_DIR / "settings", "w") as f:
-            print(f"dummy=true", file=f)
+            print(f"load_aware_scale_test=true", file=f)
 
         with open(env.ARTIFACT_DIR / "config.yaml", "w") as f:
             yaml.dump(config.ci_artifacts.config, f, indent=4)
 
+        run.run("./run_toolbox.py cluster reset_prometheus_db")
+
         failed = True
         try:
-            run.run("./run_toolbox.py cluster reset_prometheus_db")
 
-            logging.info("Waiting 5 minutes to capture some metrics in Prometheus ...")
-            time.sleep(5 * 60)
+            run.run("./run_toolbox.py load_aware scale_test")
```

Comment:
hum, I knew something was wrong :) 
You need `from_config` to run you command the settings from `command_args.yaml`
```suggestion
            run.run("./run_toolbox.py from_config load_aware scale_test")
```
otherwise, only this ^^^ command is executed, as shown by [this file](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/810/pull-ci-openshift-psap-ci-artifacts-main-load-aware-light/1674800065614450688/artifacts/light/test/artifacts/000__load_aware_scale_test/001__loadaware__scale_test/_python.cmd)

and 
```
  scale_test:
    distribution: null
    duration: null
    instances: null
```
this won't work, because it will pass the value `None` to the Python call, which isn't the same as `not passing a parameter` (only in this case the Python default value is used)

Reply:
Ah ok, I didn't realize how `from_config` worked entirely. I'll add the correct defaults to the config.